src/cmd/terraform.go

```diff
@@ -307,35 +307,55 @@ func exportServices(c *opslevel.Client, shell *os.File, directory string) {
 	}
 }
 
+func getMembershipsAsTerraformConfig(members []opslevel.TeamMembership) string {
+	membersBody := strings.Builder{}
+	for _, member := range members {
+		memberConfig := `
```

Comment:
Can `memberConfig` be moved out of the for loop?
Overall this looks good! 👍

Reply:
Moved!